drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -428,6 +448,62 @@
     outline: 2px dotted white;
 }
 
+@media screen and (max-width: 1001px) {
+    .xblock--drag-and-drop .popup {
+        background-color: rgba(255, 255, 255, 1);
+        border-radius: 10px;
+        text-align: center;
+        box-shadow: 0 0 100px rgba(0,0,0,0.4);
+    }
+
+    .xblock--drag-and-drop .popup .close-feedback-popup-button {
+        display: none;
+    }
+
+    .xblock--drag-and-drop .popup.popup-incorrect {
+        background-color: rgba(255, 255, 255, 1);
+    }
+
+    .xblock--drag-and-drop .popup .popup-header-icon {
+        display: inline;
+        margin-top: 8px;
+        color: #629a2c;
+        font-size: 12pt;
+    }
+
+    .xblock--drag-and-drop .popup .popup-header-icon.popup-header-icon-incorrect {
+        color: #ff143e;
+    }
+
+    .xblock--drag-and-drop .popup .popup-header-text {
+        display: inherit;
+    }
+
+    .xblock--drag-and-drop .popup .popup-content {
+        color: #629a2c;
+        margin: 10px 0px 0px 0px;
+        font-size: 14px;
+        background-color: #f4ffe7;
+        padding: 20px;
+    }
+    
+    .xblock--drag-and-drop .popup .popup-content.popup-content-incorrect {
+        color: #ff143e;
+        background-color: #ffe8ec;
+    }
+
+    .xblock--drag-and-drop .popup .close-feedback-popup-mobile-button {
+        display: inline;
+        cursor: pointer;
+        margin-top: 10px;
+        margin-bottom: 10px;
+        color: #3384ca;
+        font-size: 12pt;
```

Comment:
Also add `padding: 10px;` - this increases the touch sensitive area so it's easier to hit on mobile and also looks nicer IMHO.